### ISSUE TITLE
The question about weighted_CTCloss

### ISSUE THREAD

**Issue by BigTree765** [kartikgill/Easter2#9]
def ctc_custom(args):
    """
    custom CTC loss
    """
    y_pred, labels, input_length, label_length = args
    ctc_loss = K.ctc_batch_cost(
        labels, 
        y_pred, 
        input_length, 
        label_length
    )
    p = tensorflow.exp(-ctc_loss)
    gamma = 0.5
    alpha=0.25 
    return alpha*(K.pow((1-p),gamma))*ctc_loss

how to understand the codes

**Comment by kartikgill** [comment]
Hey @yusirhhh ,

Thanks for checking out Easter.

This is a weighted version of CTC loss, that helps if fast convergence of model. It also helps in achieving better accuracy when you have limited data samples per class. 
This setup is purely experimental...please feel free to change few values and do experiment. If you get some interesting results, please share with everyone here. Thanks.
